pyproject.toml

```diff
@@ -18,6 +18,8 @@ pyyaml = "^5.3.1"
 cryptography = "^3.3.1"
 django = "^3.1.4"
 boto3 = "^1.17.27"
+PyGithub = "^1.54.1"
+PyNaCl = "^1.4.0"
```

Comment:
how did you add these? It didn't update the lock file, so I assume you added these manually.

You should do it like `poetry add PyNaCl` so that it also updates the lock file

Reply:
I ended up having to delete the lock file and re-run poetry install to resolve the cryptography errors. Which ended up making many changes to the lock file.

Reply:
Just added second commit with update to `poetry.lock`

Reply:
generally blowing away the lock file is bad, but this library is so small I'm not worried about it here